sylvia-derive/src/utils.rs

```diff
@@ -97,9 +120,13 @@ pub fn as_where_clause(where_predicates: &[&WherePredicate]) -> Option<WhereClau
     }
 }
 
-pub fn emit_bracketed_generics<Generic: ToTokens>(unbonded_generics: &[&Generic]) -> TokenStream {
+pub fn emit_bracketed_generics(unbonded_generics: &[impl ToTokens]) -> TokenStream {
```

Comment:
This method has to work with `GenericParam`, `GenericArguments` and `Ident` in case of associated types so we have to keep it generic. I will revert to previos version to name the use case of the type we expect.